console/pom.xml

```diff
@@ -36,6 +36,7 @@
         <spring-framework-for-server.version>5.3.37</spring-framework-for-server.version>
         <snakeyaml-for-server.version>2.0</snakeyaml-for-server.version>
         <tomcat-embed.version>9.0.90</tomcat-embed.version>
+        <NODE_DOWNLOAD_URL>https://mirrors.aliyun.com/nodejs-release/</NODE_DOWNLOAD_URL>
```

Comment:
Thank you very much for your PR submission and attention to this issue. However, according to the discussion in the issue, it is not recommended to directly change this place globally, and there may be network problems for foreign users to connect to this address. It is suggested that local issues be solved locally.

Reply:
So do I need to close this PR？

Reply:
Yes. I look forward to your better fix.